unitylibs/core/workflow/workflow-acrobat/limits.json

```diff
@@ -65,5 +65,16 @@
       "message/rfc822",
       "text/plain"
     ]
+  },
+  "page-limit-500": {
+    "pageLimit": {
+      "maxNumPages": 500
+    }
+  },
+  "sendforsignature": {
+    "maxFileSize": 5242880,
+    "pageLimit": {
+      "maxNumPages": 25
```

Comment:
@sharmeeatadobe Have you received any additional confirmation that the limit is indeed 25 max pages? Last I heard, this was still an open question. I haven't yet received any confirmation myself. I think we can keep this at 25 for now as that is what is indicated in the current requirements, but would you be able to follow up on this in the #temp-unity-acrobat-devs channel?